README.md

```diff
@@ -419,6 +420,29 @@ The option `--seed` fixes the random generator seed. It should not
 generally be used except to reproduce a particular Bitmask
 trace.
 
+### Cancellation Backend (libinterflop_cancellation.so)
+
+The Cancellation backend implements an automatic cancellation detector at
+runtime. It is founded on bitwise operations to detect cancellation faster than
```

Comment:
This is incorrect. bitwise operations are not used to detect cancellation; we use difference in exponents.

Reply:
My bad. I was pretty sure that we use some operations on the bit representation of floating numbers to get their exponents and that I can call that bitwise operation but in any case I made a bad translation. But now I have a trouble to understand the code to get the exponent... 
`inline int32_t _get_exponent_binary64(const double d) {
  binary64 x = {.f64 = d};
  /* Substracts the bias */
  return x.ieee.exponent - DOUBLE_EXP_COMP;
}`
In this code or respectively the function for float case and 128bits float I don't understand the `binary64 x = {.f64 = d};` part. Did you have some references with whom I can understand what is happening here ?

Reply:
You did use bitwise operations actually; but I changed that to use #152 macros.

* binary64 is a union between a double and a struct with a set of bitfields corresponding to the sign, exponent, and mantissa.

* ```binary64 x = {.f64 = d};``` this copies the double to the union.
* ```x.ieee.exponent``` this extracts the exponent from the union. It is probably done with bit operations at the assembly level; but most backends do that already.